Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Eshop.Domain;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Eshop.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CategoryController : Controller
+    {
+        private static List<Category> Categories = new List<Category>
+        {
+            new Category("Computers"),
```

Comment:
OMG - did you even build the solution? Because here you are missing 2 params.
You can not even run app because of this.

Reply:
fixed, same as ProductController.cs